vcsclient/github.go

```diff
@@ -573,24 +574,21 @@ func (client *GitHubClient) DownloadFileFromRepo(ctx context.Context, owner, rep
 	body, response, err := ghClient.Repositories.DownloadContents(ctx, owner, repository, path, &github.RepositoryContentGetOptions{Ref: branch})
```

Comment:
let's change it to bitbacketResponse / githubResponse / gitlabResponse for all following changes: